modules/oe_whitelabel_helper/src/Plugin/Field/FieldFormatter/ExternalLinkFormatter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_whitelabel_helper\Plugin\Field\FieldFormatter;
+
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\link\Plugin\Field\FieldFormatter\LinkFormatter;
+
+/**
+ * Plugin implementation of the 'external_link' formatter.
+ *
+ * @FieldFormatter(
+ *   id = "external_link",
+ *   label = @Translation("Force external links to open in new tab"),
+ *   field_types = {
+ *     "link"
+ *   }
+ * )
+ */
+class ExternalLinkFormatter extends LinkFormatter {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function viewElements(FieldItemListInterface $items, $langcode) {
+    $element = parent::viewElements($items, $langcode);
+
+    foreach ($items as $delta => $item) {
+      $url = $this->buildUrl($item);
+
+      if ($url->isExternal()) {
+        $element[$delta]['#options']['attributes']['target'] = '_blank';
```

Comment:
I just found these two modules.
https://www.drupal.org/project/external_links_new_tab
https://www.drupal.org/project/sel

One interesting thing from the module pages:

> If you simply just have a target="_blank" on outbound links, you put your site visitors at risk of tabnapping.

> Additionally, it also adds rel="noopener" to external links to improve performance and prevent security vulnerabilities.

On the other hand, here I find this https://mathiasbynens.github.io/rel-noopener/

> 2021 update: Browsers now implicitly set rel=noopener for any target=_blank link, following a spec change. If the demo on this page no longer seems scary, congratulations — you’re using a modern browser!

Reply:
Let's keep `_blank` for now, and do a follow-up if we really need to add `noopener`.